skyvern-frontend/package-lock.json

```diff
@@ -52,6 +52,7 @@
       "version": "5.2.0",
       "resolved": "https://registry.npmjs.org/@alloc/quick-lru/-/quick-lru-5.2.0.tgz",
       "integrity": "sha512-UrcABB+4bUrFABwbluTIBErXwvbsU/V7TZWfmbgJfbkwiBuziS9gxdODUyuiecfdGQ85jglMW6juS3+z5TsKLw==",
+      "dev": true,
```

Comment:
@msalihaltun Please review the package-lock.json change to see if there's any impact here introducing `"dev": true`

Reply:
This doesn't seem right. We should exclude this file from the PR.